integration-tests/api/__tests__/admin/draft-order/draft-order.js

```diff
@@ -585,7 +582,7 @@ describe("/admin/draft-orders", () => {
           adjustments: expect.arrayContaining([
             expect.objectContaining({
               item_id: lineItem1.id,
-              amount: discountAmount * lineItem1WeightInTotal,
+              amount: 444.4444444444444, // discountAmount * (line item amount / amount),
```

Comment:
non-blocking: maybe we could use [expect.closeTo](https://jestjs.io/docs/expect#expectclosetonumber-numdigits) if our jest version supports it

Reply:
that is nice,  did not know about this one 😂

Reply:
There is no close to when expecting here. Only at the higher level `expect(x).toBeCloseTo`

Reply:
probably the version we're using doesn't support it yet

Reply:
I exctrated it at the top level when needed 👍